.github/workflows/gravity.yml

```diff
@@ -0,0 +1,22 @@
+name: Gravity
+on:
+  push:
+    branches:
+      - main
+  pull_request: {}
+
+jobs:
+  build:
+    name: Run Gravity
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v3
+        with:
+          version: 9
+      - run: pnpm i --frozen-lockfile
+      - run: cd packages/svelte && pnpm run build
+      - run: npx run gravityci "packages/svelte/dist/index.js"
```

Comment:
```suggestion
      - run: npx run gravityci "packages/svelte/dist/"
```

Reply:
Thanks @BobrImperator, I must have misread the gravity docs as requiring a single entry to the app to compare

Reply:
💅🏽 If the project uses `pnpm` maybe is more convenient to use [`pnpx`](https://pnpm.io/es/cli/dlx) instead?

Reply:
In my opinion since the dependency is installed already, the command should say `pnpm gravityci ..`

Reply:
> In my opinion since the dependency is installed already, the command should say `pnpm gravityci ..`

Oh yes, if it's installed already yes.

Reply:
Thanks guys, the CI seems to be working now 👍